File: src/main/java/com/haozileung/web/service/dictionary/IDicTypeService.java
package com.haozileung.web.service.dictionary;

import com.haozileung.infra.pager.PageResult;
import com.haozileung.web.domain.dictionary.DicType;


public interface IDicTypeService {

    Long save(DicType dic);

    Integer update(DicType dic);

    Integer delete(DicType dic);

    PageResult<DicType> query(DicType dic, int pageNo, int pageSize);
}

File: src/main/kotlin/com/haozileung/web/service/dictionary/IDicTypeService.kt
package com.haozileung.web.service.dictionary;

import com.haozileung.infra.pager.PageResult;
import com.haozileung.web.domain.dictionary.DicType;


interface IDicTypeService {

    fun save(dic: DicType): Long;

    fun update(dic: DicType): Int;

    fun delete(dic: DicType): Int;

    fun query(dic: DicType, offset: Int = 0, limit: Int = 10): PageResult<DicType>;
}